### DISCUSSION TITLE
Navigation bar and categories

### DISCUSSION THREAD

**Discussion by dhakelila** [Vizzuality/mangrove-atlas#294] [General]
Hello Kate and Lammert, 

I've been catching up with the team and it seems that you liked the solution Dani presented but you wanted to check with the rest of your team before approving the solution. 

When would you be able to send us your team's feedback? 

We would like to have it as soon as possible to start coding the solution. 

All the best,

**Comment by dhakelila** [comment]
[Link to the prototype that Dani prepared for your evaluation](https://basecamp.com/1756858/projects/16449126/messages/96891614)

**Comment by klongleywood** [comment]
Hi all, 

Here are our notes on the mock-up from our call today:

- Categories look good but can we change "Summary" to "Distribution and Change"
- Is there a way to completely minimize and/or narrow the widgets? -- on a narrow screen they take up too much space.  If you narrow them, we can take away the hide/download/info so you can't see those options until they are expanded. Would also be good to have an option to completely minimize them. 
- Option to turn the Protect Areas **only** when you're zoomed into the country level
- For symbols, suggest a mangrove from GMA as first category, thermometer for climate -- open to suggestions for others (fish and a wave for ecosystem services?)

**Comment by dhakelila** [comment]
Hi Kate, let see your points one by one: 

_- Categories look good but can we change "Summary" to "Distribution and Change"_
Sure, no problem. 

 _- Is there a way to completely minimize and/or narrow the widgets? -- on a narrow screen they take up too much space. If you narrow them, we can take away the hide/download/info so you can't see those options until they are expanded. Would also be good to have an option to completely minimize them._
I understand that the issue is that you don't see the full list on a small screen, isn't it? 
I'd say that once we group the widgets by category, you won't have that issue. But let's discuss this during the meeting as I'm not sure I'm fully understanding the issue. @danicaso-vizzuality , I'd say you have a say on this as well. 

_- Option to turn the Protect Areas only when you're zoomed into the country level_
This is feasible but it's not related with this task and I should check if this is included on the current scope as it requires a good amount of work. 

_- For symbols, suggest a mangrove from GMA as first category, thermometer for climate -- open to suggestions for others (fish and a wave for ecosystem services?)_
Sure, no problem!

**Comment by lyubovpencheva** [comment]
This is an old one.
